content/webapp/views/components/ImageGallery/index.tsx

```diff
@@ -260,7 +260,13 @@ const ImageGallery: FunctionComponent<{ id: string } & Props> = ({
                   ref={openButtonRef}
                   ariaControls={id}
                   ariaExpanded={isActive}
-                  dataGtmTrigger={isActive ? undefined : 'show_image_gallery'}
+                  dataGtmProps={
+                    isActive
```

Comment:
not sure about this, or if we handle `undefined` inside the object, because e.g. we might want another data-gtm-attribute here even if `isActive` is false

Reply:
Is your question "do we want to allow `undefined` versus only having that attribute there IF `isActive` is true"?